manifests/database/postgresql.pp

```diff
@@ -7,7 +7,9 @@
 
   $password = $foreman::db_password ? {
     'UNSET' => false,
-    default => postgresql::postgresql_password($foreman::db_username, $foreman::db_password),
+    # as soon, as the Function accepts Sensitive (Pull-Request pending), we can remove the Workaround
```

Comment:
https://github.com/puppetlabs/puppetlabs-postgresql/pull/1279

Reply:
The puppetlabs-postgresql PR was merged and released so this can be updated.

Reply:
I removed the Workaround and amended the Commit.